pkg/asset/cluster/tfvars.go

```diff
@@ -286,6 +286,12 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			awsMP.Set(mp.Platform.AWS)
 			masterIAMRoleName = awsMP.IAMRole
 		}
+		apiIntLBName := ""
+		apiLBName := ""
+		if installConfig.Config.AWS.UserConfiguredDNSLB.UserDNS == "Enabled" {
```

Comment:
@sadasu per the code here, only BYO LB is not in this feature's scope yet, right?